source/docs/software/kinematics-and-odometry/swerve-drive-kinematics.rst

```diff
@@ -236,6 +236,27 @@ By recording a set of swerve module states using :ref:`NetworkTables <docs/softw
         }
       };
 
+   .. code-block:: python
+
+      from wpiutil import wpistruct
+      import ntcore
+
+      class MyStruct:
+        x: int
+        y: bool
+        z: wpistruct.double
+
+      # get the default instance of NetworkTables
+      nt = ntcore.NetworkTableInstance.getDefault()
+
+      # Start publishing an array of module states with the "/SwerveStates" key
+      topic = nt.getStructArrayTopic("/SwerveStates", SwerveModuleState.struct)
```

Comment:
`.struct` isn't needed, should be `topic = nt.getStructArrayTopic("/SwerveStates", SwerveModuleState)`